src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -293,6 +293,14 @@ private static double accumulate(Double a, Double b) {
             return new JsonObject();
           }
 
+          /*
+           * Ignoring not found error from storage because after MODORDSTOR-52 changes mod-orders-storage returns 404 instead of 204
```

Comment:
I take it this is a product of switching to PgUtil?

Reply:
That's right.